extensions/amp-date-picker/amp-date-picker.md

```diff
@@ -0,0 +1,307 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-date-picker"></a> `amp-date-picker`
+
+[TOC]
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Provides a widget to select dates. The date picker can render as an overlay relative to input fields, or as a static calendar widget.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-date-picker" src="https://cdn.ampproject.org/v0/amp-date-picker-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>Size-defined layouts for <code>static</code> mode. <code>container</code> layout for <code>overlay</code> mode.</td>
+  </tr>
+</table>
+
+## Behavior
+
+The `amp-date-picker` provides two modes to render the date picker:
+
+* `mode="static"`: renders a static calendar view.
+* `mode="overlay"`: shows only when the user interacts with its input elements.
+
```

Comment:
Since mode is not required, can you add a line here saying what it defaults to if not specified? E.g. 

> `mode="static"`: renders a static calendar view. This is the default mode if no value for `mode` is specified.

Reply:
Actually, since you discuss this API below, why is it necessary to call out `mode` especially in the beginning here?

Reply:
I added it below.